src/test/java/com/algorand/algosdk/unit/AlgodPaths.java

```diff
@@ -63,4 +64,33 @@ public void getAssetByID(Long id) {
     public void getApplicationByID(Long id) {
         ps.q = algodClient.GetApplicationByID(id);
     }
+
+    @When("we make an Account Application Information call against account {string} applicationID {int}")
+    public void we_make_an_account_application_information_call_against_account_application_id(String string, Integer int1) {
+        try {
+            ps.q = algodClient.AccountApplicationInformation(new Address(string), (long)int1.intValue());
+        } catch (NoSuchAlgorithmException e) {
+            e.printStackTrace();
+        }
+    }
+    
+    @When("we make an Account Asset Information call against account {string} assetID {int}")
+    public void we_make_an_account_asset_information_call_against_account_asset_id(String string, Integer int1) {
+        try {
+            ps.q = algodClient.AccountAssetInformation(new Address(string), (long)int1.intValue());
+        } catch (NoSuchAlgorithmException e) {
+            e.printStackTrace();
+        }
+    }
+    
+    @When("we make an Account Information call against account {string} with exclude {string}")
+    public void we_make_an_account_information_call_against_account_with_exclude(String string, String string2) {
+        try {
+            AccountInformation aiq = algodClient.AccountInformation(new Address(string));
+            if (TestingUtils.notEmpty(string2)) aiq.exclude(QueryMapper.getExclude(string2));
```

Comment:
I added a comment to the test cases PR. We should add a way to provide multiple exclude values.